libs/dal/src/repositories/notification-template/notification-template.schema.ts

```diff
@@ -68,6 +68,7 @@ const notificationTemplateSchema = new Schema<NotificationTemplateDBModel>(
           type: Schema.Types.Boolean,
           default: false,
         },
+        uuid: Schema.Types.String,
```

Comment:
Just a question regarding future, are we going to search or order by uuid in the future in a way that we would need to do indexation?

Reply:
So I do not have an exact answer to this question, thinking of digest stuff maybe this will be used to enable us to handle two digest step in the same workflow...

Reply:
I was thinking that an UUID v4 might be not the best option for distributed systems and indexation. But maybe a problem for future us. :-)